site/content/docs/main/tilt.md

```diff
@@ -141,6 +148,26 @@ to learn what field/value pairs are required for your provider's credentials. Th
 
 There is a sample credentials file properly formatted for a MinIO storage credentials in `velero/tilt-resources/examples/cloud`.
 
+### Configure debugging with Delve
+If you would like to debug the Velero process, you can enable debug mode by setting the field `enable_debug` to `true` in your `tilt-resources/tile-settings.json` file.
```

Comment:
Could this flag also enable debug logging, or is that too much noise do you think?

Reply:
It could, but this is also controlled in the deployment yaml file that is used by Tilt. Perhaps another option would be to instead have a "log_level" option in the tilt settings so that it can be managed in the same file, but can be configured independently.

Reply:
Putting a separate logging flag in works for me. I'd hazard to say debug logging should be on by default in Tilt setups, but I also think that discussion might be better suited elsewhere.